### PR TITLE
Issue 2282

### PR DESCRIPTION
**Pull request by rppawlo**
fix warnings for issue #2282

### CODE REVIEW

File: packages/panzer/mini-em/example/BlockPrec/main.cpp [hunk 1]
```diff
@@ -642,12 +642,12 @@ void createExodusFile(const std::vector<Teuchos::RCP<panzer::PhysicsBlock> >& ph
...
-    Teuchos::ParameterList& cell_avg_q = output_pl.sublist("Cell Average Quantities");
+    // Teuchos::ParameterList& cell_avg_q = output_pl.sublist("Cell Average Quantities");
     Teuchos::ParameterList& cell_avg_v = output_pl.sublist("Cell Average Vectors");
     cell_avg_v.set(block_names[0],"CURRENT");
-    Teuchos::ParameterList& cell_q = output_pl.sublist("Cell Quantities");
-    Teuchos::ParameterList& nodal_q = output_pl.sublist("Nodal Quantities");
-    Teuchos::ParameterList& a_nodal_q = output_pl.sublist("Allocate Nodal Quantities");
+    // Teuchos::ParameterList& cell_q = output_pl.sublist("Cell Quantities");
+    // Teuchos::ParameterList& nodal_q = output_pl.sublist("Nodal Quantities");
+    // Teuchos::ParameterList& a_nodal_q = output_pl.sublist("Allocate Nodal Quantities");
```
Comment (jmgate) [inline packages/panzer/mini-em/example/BlockPrec/main.cpp]: Do you want these lines commented out or just removed?  We can always get them back via git if we need to.

**Comment by allevin** [comment]
@rppawlo 
Not sure if this helps...Failing 4 Tests (on Build 295)...

The following tests FAILED:
	613 - PanzerMiniEM_MiniEM-BlockPrec_Augmentation_MPI_1 (Failed)
	614 - PanzerMiniEM_MiniEM-BlockPrec_Augmentation_MPI_4 (Failed)
	615 - PanzerMiniEM_MiniEM-BlockPrec_RefMaxwell_MPI_1 (Failed)
	616 - PanzerMiniEM_MiniEM-BlockPrec_RefMaxwell_MPI_4 (Failed)
Errors while running CTest
CTest failed. The error code was: 8

**Comment by jwillenbring** [comment]
@rppawlo Here are some test results for this pull request

[https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3335369](https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3335369)

**Comment by rppawlo** [comment]
> I would remove lines rather than comment them out, but either way is fine by me.

@jmgate - Good suggestion. In the future I will just remove when possible. Turns out in this case we couldn't remove them. We needed to instantiate the sublists for validation after all. Made a minor fix and tests are all passing. Thanks!

**Comment by jmgate** [comment]
Yeah I noticed that.  Thanks for taking care of this @rppawlo.

**Pull request merged**